src/main/resources/project/server/deployApp.pl

```diff
@@ -68,9 +90,39 @@ sub main {
         $command .= ' ' . $params->{additional_options} . ' ';
     }
 
+    ######################
+    # generate possible job step summary in case of successful completion
+    ######################
+
+    # expected source - filepath or url (url without '--url=' anchor)
+    my $expected_source = $params->{warphysicalpath};
+    if ($source_is_url) {
+        $expected_source =~ s/^(--url=)(.*)$/$2/;
```

Comment:
Are you sure that greedy .* is required there? Also, are you sure, that --url will be the last parameter?

Reply:
according to the documentation:
in case of '--url' (for EAP 7 and later) - pass URL at which the deployment content is available.
NOTE: The expected format in this case should include prefix anchor '--url=', e.g. '--url=https://my.site.com/hello-world.war'.

so this is not expected to be additional option.

FYI this line of code is used only for generation of job step summary.

Reply:
So it means that user can provide a few options within "Application Content Source Path" and it can even work - but it is not expected case - all expected cases are listed in help doc. In this unexpected case which may work, a job step summary will look strange. I suggest to not cover such cases and to not document them as well.

Reply:
simplified substitution regex